src/application/extension-popup-menu/app.tsx

```diff
@@ -0,0 +1,55 @@
+import { Route, Routes } from 'react-router-dom';
+
+import { useExtensionSettings } from 'shared/extension';
+import { Closable } from 'shared/ui';
+
+import { Footer } from './footer';
+import { PopupHomeView } from './popup-home-view';
+import { TopBar } from './top-bar';
+import { CustomizationSettings } from './settings-view/customization-settings';
+import { MainSettingsMenu } from './settings-view/main-settings-menu';
+
+export const EXTENSION_POPUP_MENU_ROUTES = {
+  HOME: '/',
+  SETTINGS: {
+    MAIN_VIEW: '/settings',
+    CUSTOMIZATION: '/settings/customization',
+  },
```

Comment:
I am not sure about this nesting, usually nesting ends up in a more complex data structure than it needs to be, I suggest to do:
```ts
HOME: '/',
SETTINGS_HOME: '/settings',
SETTINGS_CUSTOMIZATION: '/settings/customization'
```